manager/manager.go

```diff
@@ -366,6 +366,10 @@ func (m *Manager) InitWithOptions(elf io.ReaderAt, options Options) error {
 		return err
 	}
 
+	for _, excludedProgram := range m.options.ExcludedProbes {
+		delete(m.collectionSpec.Programs, excludedProgram)
```

Comment:
I believe this will cause `matchSpecs` to fail because the defined `Probes` will not have a matching program in the spec.